src/tests/zf_unit/zfzockfree.c

```diff
@@ -128,10 +128,17 @@ static int create_zocket(struct zf_stack* stack, struct zf_attr* attr,
                          struct zft** tcp_out)
 {
   struct zft_handle* tcp_handle;
+  struct sockaddr_in laddr;
+  socklen_t len = sizeof(laddr);
+
   int rc = zft_alloc(stack, attr, &tcp_handle);
   if( rc < 0 )
     return rc;
 
+  zft_handle_getname(tcp_handle, (struct sockaddr *)&laddr, &len);
+  if( laddr.sin_port != 0 )
+    return -EBUSY;
```

Comment:
Why is it that `-EBUSY` was chosen here? Is it possible for a handle to still be in use (with some delayed shutdown) after we free it, so we end up here after trying to reuse it after alloc? Or is it just a useful assertion for this specific issue you're fixing? (Just curious, not saying `-EBUSY` should be something else)

Reply:
It was the second point (a useful assertion) - needed a suitable error to indicate test failure.  I wasn't sure what error was best but thought EBUSY indicated the handle was in some way still in use by still having a local address set.